react-props/react-props-and-expressions-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# react-props-and-expressions-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What are props in React?
+
+the means of passing arguments to React components when they are called
+
+- How do you declare and access props in a component?
+
+by using Type
```

Comment:
`type` is how we annotate or type our props, but how do we actually declare them? Where do the props get created? And how do we then access those props within the component they were passed to?